spec/requests/service_catalogs_spec.rb

```diff
@@ -469,6 +476,7 @@ def init_st(service_template, resource_action)
     let(:ra1)     { FactoryGirl.create(:resource_action, :action => "Provision", :dialog => dialog1) }
     let(:st1)     { FactoryGirl.create(:service_template, :name => "service template 1") }
     let(:sc)      { FactoryGirl.create(:service_template_catalog, :name => "sc", :description => "sc description") }
+    let(:run_automate_methods_on_service_catalog_api_submit) { true }
```

Comment:
Now that you fixed the `product_settings` not being used anywhere, this variable isn't being used either. I figured you meant to change line 372 (https://github.com/ManageIQ/manageiq-api/pull/485/files#diff-e129a646c1b31a559a2192293c958cf9R372) to use `product_settings` which then uses this variable.